sig-architecture/production-readiness.md

```diff
@@ -28,6 +27,19 @@ aspects of the process.
     happens if it is subsequently upgraded again?
   - Are there tests for this?
 * Scalability
+  - Does enabling/using the feautre will result in new API calls?
+    Keeping in mind the [supported limits] (e.g. 5000 nodes per cluster,
+    100 pods/s churn), what QPS is expected for each of those?
+  - Does enabling/using the feature will result in supporting new API
+    objects? How many of those will be supported (and how that translates
+    to limitations for users)?
+  - Does enabling/using the feature will result in increasing size of
+    the existing API objects? How much?
+  - Does enabling/using the feature will result in increasing latencies
+    of any operations covered by [existing SLIs/SLOs] (e.g. by adding
+    additional work)? Please describe the details about impact if so.
+  - Does enabling/using the feature will result in non-negligible increase
```

Comment:
I think it can be quite hard to assess by a person who has no experience with high scale. e.g. if someone is watching all pods, at small scale the increase can be negligible, but no-go in high scale.

 Shouldn't we ask more specific questions here: like "are you going to watch everything and do actions based on them?" and similar.

Reply:
Yeah - this one is something that I'm also not fully convinced about (i.e. I do believe this would be a useful info to have, but I'm not sure if we get useful info from KEP authors).
I'm slightly against specific questions, because either there would have to be too many of those, or we will miss too many information so that it won't be that useful.

So I'm leaning towards leaving as is for now and validating during pilot.